### DISCUSSION TITLE
How is it installed?

### DISCUSSION THREAD

**Discussion by nudefireninja** [Homebrew/discussions#4108] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 3.6.18
ORIGIN: https://github.com/Homebrew/brew
HEAD: 106af035578d5f8716ae9510c1f9ad8c19f73a79
Last commit: 5 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 34bbd048d0e948678ab4390687ff963360750368
Core tap last commit: 25 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: /usr/local/bin/mate-w
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.85.0 => /usr/bin/curl
macOS: 13.1-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

I am confused about the output of `brew list` and `brew info certbot`. It seems to indicate that `certbot` is installed, yet also that one of `certbot`'s dependencies, `dialog`, is not installed. Note that I have been using `certbot` without issues for the past several years.

It's just something I noticed and got confused by. As far as I understand, `brew list` shows installed formulae and `brew info` shows info about an installed formula.

<img width="288" alt="brew-certbot-smol" src="https://user-images.githubusercontent.com/6565746/212169513-d8494187-19a0-4309-954c-f1c1b71b1a03.png">

I would expect it to show every required dependency as installed.

**Comment by cho-m** [comment, marked as answer]
Can you check information on dependency (`brew info dialog`)?

I think that symbol may also be used if outdated. If so, may be nicer to show the another symbol instead.

**Comment by nudefireninja** [comment]
Hi, Michael!

```
➜  ~ brew info dialog 
==> dialog: stable 1.3-20221229 (bottled)
Display user-friendly message boxes from shell scripts
https://invisible-island.net/dialog/
/opt/homebrew/Cellar/dialog/1.3-20220728 (14 files, 941.4KB) *
  Poured from bottle on 2022-12-23 at 16:34:59
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/dialog.rb
License: LGPL-2.1-or-later
==> Analytics
install: 11,258 (30 days), 22,869 (90 days), 112,529 (365 days)
install-on-request: 1,982 (30 days), 3,531 (90 days), 16,746 (365 days)
build-error: 0 (30 days)
```

I also ran `brew outdated`:

```
➜  ~ brew outdated
certbot (2.1.0) < 2.2.0
dialog (1.3-20220728) < 1.3-20221229
restic (0.14.0) < 0.15.0
swiftformat (0.50.6) < 0.50.7
xz (5.4.0) < 5.4.1
```

Then I ran `brew upgrade` and after that it's showing a green ✅ for dialog in `brew info certbot`

So I think you are right, and I agree that a different symbol could be clearer.

Thanks!
